crates/fluvio-controlplane-metadata/src/spu/spec.rs

```diff
@@ -137,6 +137,8 @@ pub struct CustomSpuSpec {
     pub private_endpoint: Endpoint,
     #[cfg_attr(feature = "use_serde", serde(skip_serializing_if = "Option::is_none"))]
     pub rack: Option<String>,
+    #[cfg_attr(feature = "use_serde", serde(skip_serializing_if = "Option::is_none"))]
+    pub public_endpoint_local: Option<Endpoint>,
```

Comment:
why need to add here?

Reply:
This is used in the command to register the custom SPU

Reply:
It is using CustomSpuSpec,  don't think there is need to add to `SpuSpec`

Reply:
`SpuSpec` already has the `public_endpoint_local` field.
This PR adds the field to `CustomSpuSpec`.

Reply:
Got it. got confused about diff